code/modules/events/blob/theblob.dm

```diff
@@ -220,7 +220,6 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 			damage_amount *= brute_resist
 		if(BURN)
 			damage_amount *= fire_resist
-		if(CLONE)
```

Comment:
Yes this technically means blobs no longer take clone damage now.

However, I dont think anyone actually ever did that, and it also doesn't make a huge amount of sense.